README.md

```diff
@@ -10,60 +10,32 @@ Faker is heavily inspired by Perl's [Data::Faker](http://search.cpan.org/~jasonk
 
 Faker requires PHP >= 7.1.
 
+## Getting Started
 
-# Table of Contents
+### Installation
 
-- [Installation](#installation)
-- [Basic Usage](#basic-usage)
-- [Formatters](#formatters)
-	- [Base](#fakerproviderbase)
-	- [Lorem Ipsum Text](#fakerproviderlorem)
-	- [Person](#fakerprovideren_usperson)
-	- [Address](#fakerprovideren_usaddress)
-	- [Phone Number](#fakerprovideren_usphonenumber)
-	- [Company](#fakerprovideren_uscompany)
-	- [Real Text](#fakerprovideren_ustext)
-	- [Date and Time](#fakerproviderdatetime)
-	- [Internet](#fakerproviderinternet)
-	- [User Agent](#fakerprovideruseragent)
-	- [Payment](#fakerproviderpayment)
-	- [Color](#fakerprovidercolor)
-	- [File](#fakerproviderfile)
-	- [Image](#fakerproviderimage)
-	- [Uuid](#fakerprovideruuid)
-	- [Barcode](#fakerproviderbarcode)
-	- [Miscellaneous](#fakerprovidermiscellaneous)
-	- [Biased](#fakerproviderbiased)
-	- [Html Lorem](#fakerproviderhtmllorem)
-- [Modifiers](#modifiers)
-- [Localization](#localization)
-- [Populating Entities Using an ORM or an ODM](#populating-entities-using-an-orm-or-an-odm)
-- [Seeding the Generator](#seeding-the-generator)
-- [Faker Internals: Understanding Providers](#faker-internals-understanding-providers)
-- [Real Life Usage](#real-life-usage)
-- [Language specific formatters](#language-specific-formatters)
-- [Third-Party Libraries Extending/Based On Faker](#third-party-libraries-extendingbased-on-faker)
-- [License](#license)
-
-
-## Installation
-
-```sh
+```shell
 composer require fakerphp/faker
 ```
 
-## Basic Usage
+### Documentation
+
+Full documentation can be found over on [fakerphp.github.io](https://fakerphp.github.io).
 
-### Autoloading
+### Basic Usage
+
+#### Autoloading
```

Comment:
Let's remove this section about autoloading. 

95+% is using composer, if you really want to know about the custom autoloader then you can find it in the docs.